### DISCUSSION TITLE
Is it possible for Foam to support reStructuredText?

### DISCUSSION THREAD

**Discussion by SonnySwolf** [foambubble/foam#1078] [Ideas]
Hi all,
    In my project, I often need to use reStructured with shpinx to write documents. Is it possible for Foam to support reStructuredText?

**Comment by riccardoferretti** [comment]
Atm Foam core makes several assumptions that are tailored to markdown. Once the API is released though I can see someone writing a Provider for reST, I just can't give a timeline for that

**Comment by CarbonMan** [comment]
This would be great
